Bio/File.py

```diff
@@ -674,12 +697,16 @@ def __iter__(self):
         # Python 2, use iteritems but not items etc
         # Just need to override this...
         def keys(self):
-            """Return a list of all the keys (SeqRecord identifiers)."""
+            """Iterate over the keys.
+
+            This tries to act like a Python 3 dictionary, and does not return
+            a list of keys due to memory concerns.
+            """
             return [str(row[0]) for row in
                     self._con.execute("SELECT key FROM offset_data;").fetchall()]
 
     def __getitem__(self, key):
-        """x.__getitem__(y) <==> x[y]"""
+        """Return the item when key is specified."""
```

Comment:
Using 'when' seems odd as the key must be given.

I would suggest ``Return record for the specified key.``